src/Spout/Writer/ODS/Internal/Worksheet.php

```diff
@@ -191,27 +192,33 @@ protected function getCellXML($cellValue, $styleIndex, $numTimesValueRepeated)
             $data .= ' table:number-columns-repeated="' . $numTimesValueRepeated . '"';
         }
 
-        if (CellHelper::isNonEmptyString($cellValue)) {
+        if ($cellValue instanceof Cell) {
```

Comment:
Ok, we can refactor later. But eventually we should promote the use of `Cell` objects. If it isn't used, the scalar value should be converted to a `Cell` object as early as possible so that all internal components only work with a single type.

Maybe add a TODO so we don't forget about it?